example/exporter/kubelet-servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+# You need to have "--kubelet-object=kube-system/kubelet" as a prometheus-operator arg
```

Comment:
@ese I agree 100%! It's exactly what we do in [kube-prometheus](https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus) (our cluster-monitoring manifest collection):

https://github.com/coreos/prometheus-operator/blob/882dab40a3c0c7727ac08b0635be1b51453c2097/contrib/kube-prometheus/manifests/prometheus/prometheus-k8s-servicemonitors.yaml